modules/odo-new-commands-30.adoc

```diff
@@ -0,0 +1,68 @@
+
+Port forwarding to access the application:: The `odo dev` command uses port-forwarding for application access, making it much faster to access the application.
+
+== odo add binding
+
+The `odo add binding` command allows the application to connect to an Operator-backed service and replaces the `odo link` command.
+
+Interactive mode:: The `odo add binding` command improves the developer experience by providing an interactive mode that lists bindable service in all the namespaces, and providing a way to personalize related configuration.
+
+Ability to connect to a service in any namespace:: The `odo add binding` command allows the application to connect to any bindable service in any namespace, not just in the same namespace as the application.
+
+Run from anywhere:: You can run the `odo add binding` command from anywhere and it does not require access to a Devfile to work. With this method, you can directly create the `ServiceBinding` resource on the cluster, or you can write the YAML to a file, or display it on the command line.
+
+Work without a running application:: Compared to version 2, the `odo add binding` command does not require the application to be running on the cluster, making it even faster.
+
+Only supports bindable Operators::
+The version 2 `odo link` command links an application to another application and any Operator-backed service. However, `odo` version 3 currently only supports connecting to services backed by Operators that are considered bindable by the Service Binding Operator.
+
+Requires the Service Binding Operator:: You must install the Service Binding Operator on the cluster before running the `odo add binding` command. The command relies on the Operator to provide it with a list of bindable services.
+
+== odo deploy
+
+The `odo deploy` command allows you to run the outerloop of your development cycle. When your application is running successfully in your local environment, you can use `odo deploy` to move it to the cluster.
+
+== odo registry
+
+The `odo registry` command now lists all the Devfiles available in the Devfile registries that you have added to your system.
+
+Filtering the Devfiles:: The `odo registry` command can filter the list of Devfiles, based on a keyword by using the `--filter` flag, based on a Devfile registry by using the `--devfile-registry` flag, or based on the Devfile name by using the `--devfile` flag.
+
+Detailed information about the Devfiles:: The `odo registry` command can provide detailed information about all the Devfiles, when you run it with `--details` flag.
+
+== odo list
+
+The `odo list` command lists all the resources created by `odo`, including components and bindings.
+
+Improved  information:: 
+The `odo list` command provides a better developer experience by outputting information about your application such as the mode it is running in, what tool and version is managing it, and the project type.
```

Comment:
Principle of parallelism: s/what tool and version is managing it/the tool and version managing it